compute_sdk/globus_compute_sdk/sdk/executor.py

```diff
@@ -362,8 +362,8 @@ def register_function(
             registration upstream
         """
         if self._stopped:
-            err_fmt = "%s is shutdown; refusing to register function"
-            raise RuntimeError(err_fmt % repr(self))
+            err = f"{self!r} is shutdown; refusing to register function"
```

Comment:
I've rarely used !r, good reminder!

However, can't this be condensed to `raise RuntimeError(f"{self!r} is shutdown; refusing to register function")`
? As err is not used again (like the previous err_fmt)